client/tailscale/localclient.go

```diff
@@ -81,6 +89,31 @@ func (lc *LocalClient) socket() string {
 	return paths.DefaultTailscaledSocket()
 }
 
+// WithRequestHeaders returns a copy of the *LocalClient that will send
+// the given http headers on every request.
```

Comment:
HTTP

Reply:
Removing the whole thing per your comment above ✌🏼